UMI3D-SDK/Assets/EnvironmentDevelopmentKit/UserCapture/Runtime/Tracking/UMI3DEmbodimentManager.cs

```diff
@@ -152,15 +159,25 @@ protected void DeleteEmbodiment(UMI3DUser user)
         }
 
         /// <summary>
-        /// Load an Avatar Node.
+        /// Load an Avatar Node with an important update
         /// </summary>
-        /// <param name="node">the avatar node to load.</param>
+        /// <param name="node">the avatar node to load</param>
         public void LoadAvatarNode(UMI3DAbstractNode node)
         {
             LoadEntity op = node.Register();
-            transaction.Operations.Add(op);
+            op += UMI3DEnvironment.GetEntities<UMI3DUser>();
+
+            UMI3DServer.Dispatch(new Transaction
+            {
+                Operations = new List<Operation> { op },
+                reliable = true
+            });
```

Comment:
the loadEntity return with `LoadEntity op = node.Register();` should already contain 'UMI3DEnvironment.GetEntities<UMI3DUser>();'